components/sections/SponsorShowcase.tsx

```diff
@@ -1,16 +1,24 @@
 import { ReactElement } from "react";
 import styled from "@emotion/styled";
-import { Box, Grid, Image } from "@chakra-ui/core";
+import { Box, Grid, Image, Link } from "@chakra-ui/core";
 import Wrapper from "@/components/Wrapper";
 import { HeadlineAuto, Headline5 } from "../core/Text";
 import { LocationHashEnum, styles } from "@/src/constants";
 
 const imgPath = "/images";
 
 const SponsorImage = styled(Image)`
+  margin: 0 auto;
   user-select: none;
+`;
+
+const SponsorLink = styled(Link)`
   margin-top: 32px;
   margin-bottom: 32px;
+  &:hover,
+  &:focus {
+    transform: scale(1.1);
```

Comment:
I'm not sure about this but 10% increase in size for the larger gold sponsors is a lot more in px than for the smaller bronze ones, which can look off with the inconsistency?

Perhaps something that doesn't change size like slightly lowering opacity might work better. Thoughts?